tests/i3c_ccc_tests.py

```diff
@@ -92,5 +93,76 @@ def test_i3c_ccc_setdasa_errors(self):
                               (False,
                                {"error": "INVALID_ADDRESS", "error_data": [{"address": "0x08", "error": "ADDRESS_ALREADY_ASSIGNED_TO_I3C_DEVICE"}]}))
 
+    def test_i3c_ccc_entdaa(self):
+        if self.use_simulator:
+            self.skipTest("For real device only")
+
+        self.i3c.init_bus(3300)
+        (deviceFound, bmm323) = self.i3c.find_target_device_by_pid(BMI323_DATA["asString"])
+        if not deviceFound:
+            self.skipTest("For BMM323")
+
+        entdaa_target_dict = {
+            "bmm323" : {
+            "staticAddress" : 0x14,
+            "dynamicAddress" : 0x09,
+            "i3cFeatures": 11,
+            "maxIbiPayloadLength": 233,
+            "bcr" : 6,
+            "dcr" : 239,
+            "pid" : [7, 112, 16, 67, 16, 0]
+            },
+        }
+
+        (success, response) = self.i3c.ccc_getpid(bmm323["dynamic_address"])
+        self.assertTupleEqual((True, BMI323_DATA["asInt"]), (success, response))
+
+        (res, msg) = self.i3c.reset_bus()
+        self.assertTupleEqual((True, 3300), (res, msg))
+
+        (res, msg) = self.i3c.ccc_entdaa(entdaa_target_dict)
+        self.assertTupleEqual((True, None), (res, msg))
+        
+        (success, response) = self.i3c.ccc_getpid(0x09)
+        self.assertTupleEqual((True, BMI323_DATA["asInt"]), (success, response))
+
+        (success, response) = self.i3c.targets()
+        self.assertTupleEqual((True, {
+                "static_address" : 0x14,
+                "dynamic_address" : 0x09,
+                "bcr" : 6,
+                "dcr" : 239,
+                "pid" : ["0x07", "0x70", "0x10", "0x43", "0x10", "0x00"]
+                },), (success, response[0]))
+    
```

Comment:
Apparently, due to the fact there are 2 devices connected to the bus (`BMM350` and `BMi323`), this test may fail since you are comparing with `response[0]`:

```
F....
======================================================================
FAIL: test_i3c_ccc_entdaa (__main__.TestSupernovaController)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\jpons\Binho\BMC\SupernovaController\tests\i3c_ccc_tests.py", line 130, in test_i3c_ccc_entdaa
    self.assertTupleEqual((True, {
AssertionError: Tuples differ: (True[16 chars]ss': 20, 'dynamic_address': 9, 'bcr': 6, 'dcr'[59 chars]0']}) != (True[16 chars]ss': 0, 'dynamic_address': 8, 'bcr': 38, 'dcr'[58 chars]0']})

First differing element 1:
{'static_address': 20, 'dynamic_address': 9, 'bcr': 6, 'dcr'[58 chars]00']}
{'static_address': 0, 'dynamic_address': 8, 'bcr': 38, 'dcr'[57 chars]00']}

  (True,
-  {'bcr': 6,
?          ^

+  {'bcr': 38,
?          ^^

-   'dcr': 239,
?          ^^^

+   'dcr': 67,
?          ^^

-   'dynamic_address': 9,
?                      ^

+   'dynamic_address': 8,
?                      ^

-   'pid': ['0x07', '0x70', '0x10', '0x43', '0x10', '0x00'],
?                                      ^       ^

+   'pid': ['0x07', '0x70', '0x10', '0x33', '0x00', '0x00'],
?                                      ^       ^

-   'static_address': 20})
?                     -

+   'static_address': 0})

----------------------------------------------------------------------
Ran 5 tests in 3.297s

FAILED (failures=1)
```

We printed `self.i3c.targets()` obtaining:
`(True, [{'static_address': 0, 'dynamic_address': 8, 'bcr': 38, 'dcr': 67, 'pid': ['0x07', '0x70', '0x10', '0x33', '0x00', '0x00']}, {'static_address': 0, 'dynamic_address': 9, 'bcr': 6, 'dcr': 239, 'pid': ['0x07', '0x70', '0x10', '0x43', '0x10', '0x00']}])`

Offline we discussed that a solution to this little issue could be filter the devices by pid and after that making the assertion.